webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_team_edition.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+
+import Logo from 'components/common/svg_images_components/logo_dark_blue_svg';
+
+const ProductBrandingTeamEditionContainer = styled.div`
+    display: flex;
+    align-items: center;
+    
+    svg,
+    svg path {
+        fill: rgba(var(--sidebar-text-rgb), 0.75);
+    }
+    > * + * {
```

Comment:
Again here this confusing CSS can be avoided by styling the individual components.

Reply:
This comes from the existing code in `product_branding`, I did want to replicate the behavior introducing the minimal amount of difference.